core/src/main/java/com/datastrato/gravitino/storage/kv/KvEntityStore.java

```diff
@@ -199,72 +189,15 @@ public <E extends Entity & HasIdentifier> E update(
 
           // Update the name mapping
           nameMappingService.updateName(
-              generateKeyForMapping(ident), generateKeyForMapping(updatedE.nameIdentifier()));
+              generateKeyForMapping(ident, entityType, nameMappingService),
+              generateKeyForMapping(updatedE.nameIdentifier(), entityType, nameMappingService));
```

Comment:
Only changing here is enough, do you need to change put/get also?

Reply:
all have been changed through method `concatIdAndName` and `put/get/update` will use this method to generate key format.